index.js

```diff
@@ -0,0 +1,95 @@
+const express = require('express');
+const knex = require('knex');
+
+const knexConfig = require('./knexfile.js');
+
+const server = express();
+
+server.use(express.json());
+
+// connect to the database
+const db = knex(knexConfig.development);
+
+server.get('/', (req, res) => {
+  res.send('api working');
+});
+
+
+server.get('/api/cohorts', (req, res) => {
+  db('cohorts')
+    .then(cohorts => {
+      res.status(200).json(cohorts);
+    })
+    .catch(err => res.status(500).json(err));
+});
+
+server.get('/api/cohorts/:id', (req, res) => {
+  db('cohorts')
+    .where({ id: req.params.id })
```

Comment:
Another option for getting this id here would be nested destructuring in the parameter list:
```javascript
server.get('/api/cohorts/:id', ({ params: { id } }, res) => {
  db('cohorts')
    .where({ id })
    // ...
```
and one way to format this so that the first line isn't too long:
```javascript
server.get('/api/cohorts/:id', (
  { params: { id } }, res
) => {
  db('cohorts')
    .where({ id })
    // ...
```